CMakeLists.txt

```diff
@@ -17,23 +17,37 @@ cmake_minimum_required(VERSION 3.12)
 # Command line argument takes precedence over environment variable.
 # Loading this before project definition to allow setting the compiler.
 if (MACHINE_CFG)
-  if(EXISTS "${MACHINE_CFG}")
-    include(${MACHINE_CFG})
-  else()
-    message(FATAL_ERROR "Given machine configuration at "
-      "${MACHINE_CFG} not found.")
-  endif()
+  set(USE_MACHINE_CFG ${MACHINE_CFG})
 elseif (DEFINED ENV{MACHINE_CFG})
-  if(EXISTS "$ENV{MACHINE_CFG}")
-    include($ENV{MACHINE_CFG})
-  else()
-    message(FATAL_ERROR "Given machine configuration from environment variable "
-      "MACHINE_CFG at $ENV{MACHINE_CFG} not found.")
+  set(USE_MACHINE_CFG $ENV{MACHINE_CFG})
+endif()
```

Comment:
I think this logic should probably be used on a lot of the variables we set throughout our cmake config.

Reply:
Talked offline - this can be resolved later.